.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
What is this file doing?

Reply:
I ran debug inside of vscode to see what it does forgot to remove it removing it now